docs/source/llm/run-with-c-plus-plus.md

```diff
@@ -0,0 +1,240 @@
+# Running LLMs with C++
```

Comment:
Overall I would add a section like this https://github.com/pytorch/executorch/blob/main/examples/models/llama/README.md#step-3-run-on-your-computer-to-validate so the user can just use the C++ runner as a binary too